src/Components/PostComment.jsx

```diff
@@ -0,0 +1,52 @@
+import { useContext, useState } from "react"
+import { postComment } from "../utils/api"
+import { CurrentUserContext } from "../Contexts/CurrentUser"
+import { useParams } from "react-router-dom"
+
+const PostComment = ({setComments}) => {
+    const [commentInput, setCommentInput] = useState("")
+    const [error, setError] = useState(null)
+    const {currentUser} = useContext(CurrentUserContext)
+    const params = useParams();
+    const reviewId = params.review_id
+
+    const handleInput = (event) => {
+        setCommentInput(event.target.value)
+    }
+    
+    const handlePostComment = (event) => {
+        event.preventDefault()
+        if (!currentUser) {
+            setError("must be logged in")
+        } else {
+        setError("")
+        setComments(currentComments => {
+            return [...currentComments, {
```

Comment:
If you want a new comment to appear at the top of your comments, double check the order here